lib/activerecord-import/import.rb

```diff
@@ -476,7 +476,7 @@ def import_with_validations( column_names, array_of_attributes, options = {} )
           next if model.valid?(options[:validate_with_context])
           raise(ActiveRecord::RecordInvalid, model) if options[:raise_error]
           array_of_attributes[i] = nil
-          failed_instances << model.dup
```

Comment:
See https://github.com/zdennis/activerecord-import/blob/master/lib/activerecord-import/import.rb#L473. The variable `model` is instantiated outside of the loop so it is being reused to save the time of repeatedly instantiating a new one.

If you don't dupe it, `failed_instances` will be an array of the same object.